webpack.config.js

```diff
@@ -29,7 +29,9 @@ const WORDPRESS_NAMESPACE = '@wordpress/';
 
 const gutenbergPackages = Object.keys( dependencies )
 	.filter( ( packageName ) => packageName.startsWith( WORDPRESS_NAMESPACE ) )
-	.map( ( packageName ) => packageName.replace( WORDPRESS_NAMESPACE, '' ) );
+	.map( ( packageName ) => packageName.replace( WORDPRESS_NAMESPACE, '' ) )
+	// exclude react-native packages from the main build
+	.filter( ( packageName ) => ! packageName.startsWith( 'react-native' ) );
```

Comment:
In the meantime, @youknowriad introduced a list of bundled packages that don't get included:

https://github.com/WordPress/gutenberg/blob/feed14ac384266b0ffe0740e1e99f0eb7d0c30f7/webpack.config.js#L29-L37

Closer to merge, we should consolidate it somehow.